predictor_pol/templates/success.html

```diff
@@ -12,20 +12,80 @@
     <span class="navbar-brand mb-0 h1">aquienvoto.uy</span>
 </nav>
 <div class="container">
-    <div class="alert alert-success thank-you" role="alert">
+    <div class="alert alert-success thank-you mb-5" role="alert">
         Gracias por completar el cuestionario.
     </div>
 
-    <h2>Opcional</h2>
-    <p>¿Deseas dejarnos tu e-mail para que cuando este listo
-        el modelo podamos avisarte la predicción que hubiera hecho contigo?</p>
+    <h2 class="mb-5">El candidato recomendado es <span class="badge badge-secondary">{{predicted_candidate_name}}</span></h2>
```

Comment:
Finish the sentence with a `.`

Reply:
I'm adding '.' to everything else, but this one doesn't look good with "badge" at the end.